libvast/include/vast/concept/parseable/vast/pipeline.hpp

```diff
@@ -0,0 +1,49 @@
+//    _   _____   __________
+//   | | / / _ | / __/_  __/     Visibility
+//   | |/ / __ |_\ \  / /          Across
+//   |___/_/ |_/___/ /_/       Space and Time
+//
+// SPDX-FileCopyrightText: (c) 2023 The VAST Contributors
+// SPDX-License-Identifier: BSD-3-Clause
+
+#pragma once
+
+#include "vast/concept/parseable/core/choice.hpp"
+#include "vast/concept/parseable/core/operators.hpp"
+#include "vast/concept/parseable/core/plus.hpp"
+#include "vast/concept/parseable/string/char.hpp"
+#include "vast/concept/parseable/string/char_class.hpp"
+#include "vast/concept/parseable/vast/data.hpp"
+
+#include <fmt/format.h>
+
+namespace vast::parsers {
+using namespace parser_literals;
```

Comment:
Similarly, this should never be included at the namespace level. In the case where you need it below, just use `chr{'-'}` rather than `'-'_p`.

Reply:
Removed in https://github.com/tenzir/vast/pull/2877/commits/c56df12445c4ab032cd0aca5d6d3ab8d993b619d